piu/cli.py

```diff
@@ -11,6 +11,7 @@
 import requests
 import socket
 import yaml
+import pyperclip
```

Comment:
this fails if xclip is not installed, can you please wrap it with try/except and just have the same behavior as before?